config/set-balance.js

```diff
@@ -0,0 +1,124 @@
+const path = require('path');
```

Comment:
Documentation update needed in:
- https://www.librechat.ai/docs/configuration/dotenv#balance
- https://www.librechat.ai/docs/configuration/token_usage